core/src/main/resources/jenkins/management/AdministrativeMonitorsDecorator/resources.css

```diff
@@ -96,7 +96,8 @@
     color: var(--link-color);
     text-decoration: underline;
     padding: 0;
-    font-weight: normal;
+    font-weight: 600;
```

Comment:
Is it fine to have both of them here?

Reply:
It's actually needed. Adjuncts do not go through the build pipeline and thus we need to set the fallbacks for CSS variables manually.